segmentstore/storage/impl/src/test/java/io/pravega/segmentstore/storage/impl/extendeds3/S3JerseyClientWrapper.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.extendeds3;
+
+import com.emc.object.Range;
+import com.emc.object.s3.S3Config;
+import com.emc.object.s3.bean.AccessControlList;
+import com.emc.object.s3.bean.CopyPartResult;
+import com.emc.object.s3.bean.DeleteObjectsResult;
+import com.emc.object.s3.bean.PutObjectResult;
+import com.emc.object.s3.jersey.S3JerseyClient;
+import com.emc.object.s3.request.CopyPartRequest;
+import com.emc.object.s3.request.DeleteObjectsRequest;
+import com.emc.object.s3.request.PutObjectRequest;
+import com.emc.object.s3.request.SetObjectAclRequest;
+import lombok.Synchronized;
+
+/**
+ * Wrapper over S3JerseyClient. This implements ACLs, multipart copy and multiple writes to the same object on top of S3Proxy implementation.
+ */
+public class S3JerseyClientWrapper extends S3JerseyClient {
```

Comment:
S3JerseyClient implements large number of methods which we need to implement if we inherit from S3Client directly. It is more of a convenience decision.